lib/ecto/migrator.ex

```diff
@@ -47,7 +47,7 @@ defmodule Ecto.Migrator do
   """
   @spec up(Ecto.Repo.t, integer, Module.t, Keyword.t) :: :ok | :already_up | no_return
   def up(repo, version, module, opts \\ []) do
-    versions = migrated_versions(repo)
+    versions = migrated_versions(repo, Keyword.get(opts, :prefix, nil))
```

Comment:
`Keyword.get(opts, :prefix, nil)` can be `opts[:prefix]` here and below (since it will return `nil` if there is no key)